client/me/developer/dev-survey-notice.tsx

```diff
@@ -0,0 +1,74 @@
+import { Gridicon } from '@automattic/components';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import surveyImage from 'calypso/assets/images/illustrations/developer-survey.svg';
+
+type DevSurveyNotice = {
+	onClose: ( remindTimeInSeconds: number ) => void;
+	onOk: () => void;
+	localeSlug: string;
+};
+
+const ONE_DAY_IN_SECONDS = 24 * 60 * 60;
+const ONE_YEAR_IN_SECONDS = 365 * ONE_DAY_IN_SECONDS;
+
+export const DevSurveyNotice = ( { onClose, onOk, localeSlug }: DevSurveyNotice ) => {
+	const translate = useTranslate();
+
+	const href =
+		localeSlug === 'es'
+			? 'https://wordpressdotcom.survey.fm/developer-survey-es'
+			: 'https://wordpressdotcom.survey.fm/developer-survey';
+
+	return (
+		<div className="developer-survey-notice">
+			<Button
+				className="developer-survey-notice__backdrop"
+				onClick={ () => onClose( ONE_YEAR_IN_SECONDS ) }
```

Comment:
```suggestion
				onClick={ () => onClose( ONE_SECOND_IN_SECONDS ) }
```

I worry that we're making it too easy to close the prompt now. I'd suggest bringing back to prompt after one day when it was closed by clicking the backdrop.

Reply:
`ONE_SECOND_IN_SECONDS` 😆

Reply:
Sorry :p